exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ColumnType.kt

```diff
@@ -771,7 +771,6 @@ class BlobColumnType : ColumnType() {
 
     override fun valueFromDB(value: Any): ExposedBlob = when (value) {
         is ExposedBlob -> value
-        is Blob -> ExposedBlob(value.binaryStream)
```

Comment:
Could you confirm why this branch is not needed?

Reply:
The function name is a bit misleading, since the conversion itself is done at another function.
The `Blob` from the database is read in exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/ResultRow.kt#L130 via `columnType.readObject` which creates the `ExposedBlob` from the database-object.